crowbar_framework/app/models/node_object.rb

```diff
@@ -1329,6 +1329,22 @@ def set_state(state)
     result
   end
 
+  def actions
+    [
+      "allocate",
+      "delete",
+      "identify",
```

Comment:
This is not that easy as the machines controller generates dynamic methods. I don't really want to mix that.